Apps/WebClient/src/ClientApp/src/models/credentialConnection.ts

```diff
@@ -0,0 +1,36 @@
+import { StringISODateTime } from "./dateWrapper";
+
+export interface CredentialConnection {
+    walletConnectionId: string;
+    hdid: string;
```

Comment:
We may have small tweaks to this but good for now.

Reply:
Yeah, is a good enough approximation for now